docs/recipes/endpoint-testing-with-mongoose.md

```diff
@@ -0,0 +1,281 @@
+# Endpoint testing with Mongoose
+
+This recipe shows you how to test your endpoints with AVA and Mongoose.
+
+## Setup
+
+This recipe uses the following libraries:
```

Comment:
I think you should mention here that the recipe assumes you have an Express server